### PR TITLE
Update page-help.php

### PR DESCRIPTION
**Pull request by MoritzKa**
added more information to the help page

### CODE REVIEW

File: admin/settings/class-general.php [hunk 1]
```diff
@@ -72,17 +72,17 @@ public function set_form_fields()
...
             ),
         );
 
         $fields[] = array(
             'tab'     => 'value4value',
             'field'   => array(
                 'name'  => 'v4v_node_key',
-                'label' => __('Node key', 'lnp-alby'),
-                'description' => __('Node address - the lightning node public key', 'lnp-alby'),
+                'label' => __('Node Address', 'lnp-alby'),
```
Comment (bumi) [inline admin/settings/class-general.php:84]: it's actually the public key, isn't it?

File: admin/settings/class-general.php [hunk 2]
```diff
@@ -106,7 +106,7 @@ public function set_form_fields()
             'field'   => array(
                 'name'  => 'cookie_timeframe_days',
                 'label' => __('Cookie timeframe', 'lnp-alby'),
-                'description' => __('Paid articles are saved in a cookie. How many days should these cookie be valid? (default: 180)', 'lnp-alby'),
+                'description' => __('Paid articles are saved in a cookie. How many days should these cookies be valid? (default: 180)', 'lnp-alby'),
```
Comment (bumi) [inline admin/settings/class-general.php:109]: it's only one. why plural?

Comment (MoritzKa) [reply admin/settings/class-general.php:109]: I cannot tell from a technical perspective. 
I should be either: "these cookies" or "the cookie"

File: admin/settings/class-general.php (hunk 1 above)
Comment (MoritzKa) [reply admin/settings/class-general.php:84]: Yes, I think "public key" is the very correct answer. Some websites like amboss.space use "address"
I don´t care what we use. I thought to make it more consistent and easy to understand for the user.

File: admin/settings/class-general.php (hunk 2 above)
Comment (bumi) [reply admin/settings/class-general.php:109]: true. :)

**Pull request merged**